migrations/v5-MIGRATION.md

```diff
@@ -0,0 +1,75 @@
+## API changes
+
```

Comment:
maybe we can add a comment about reporting bugs. and we probably want a section about amazon support as well right?